src/settings.cpp

```diff
@@ -1153,6 +1188,27 @@ static void SaveVersionInConfig(IniFile &ini)
 	group.GetOrCreateItem("ini_version").SetValue(std::to_string(INIFILE_VERSION));
 }
 
+/**
+ * Save BaseGraphics set selection and configuration.
+ */
+static void GraphicsSetSaveConfig(IniFile &ini)
+{
+	const GraphicsSet *used_set = BaseGraphics::GetUsedSet();
+	if (used_set == nullptr) return;
+
+	ini.RemoveGroup("graphicsset");
+	IniGroup *group = ini.GetGroup("graphicsset");
```

Comment:
`GetGroup()` will no longer create a group.

```suggestion
	ini.RemoveGroup("graphicsset");
	IniGroup *group = ini.GetGroup("graphicsset");
```